components/schema-migrator/migrations/director/20230119093834_introduce-cert-subject-mapping-entity.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE cert_subject_mapping (
+    id UUID PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
+    subject VARCHAR(256) NOT NULL,
+    CONSTRAINT cert_mapping_subject_unique UNIQUE (subject),
+    consumer_type VARCHAR(256) NOT NULL CHECK ( consumer_type IN ('Runtime', 'Integration System', 'Application', 'Super Admin"', 'Business Integration', 'Technical Client', 'global')),
```

Comment:
This is really hard-to-maintain list, and we have it in a few places in the code anyways. And adding a new consumer_type is already a hassle, now making a migration for it seems unnecessary complex.

Reply:
fixed with https://github.com/kyma-incubator/compass/pull/2878